src/main/scala/is/hail/methods/MendelErrors.scala

```diff
@@ -129,47 +134,82 @@ case class MendelErrors(trios: IndexedSeq[CompleteTrio],
   def nErrorPerNuclearFamily: RDD[((String, String), (Int, Int))] = {
     val parentsRDD = sc.parallelize(nuclearFams.keys.toSeq)
     mendelErrors
-      .map(me => ((me.trio.dad, me.trio.mom), (1, if (me.variant.altAllele.isSNP) 1 else 0)))
+      .map(me => ((me.trio.knownDad, me.trio.knownMom), (1, if (me.variant.altAllele.isSNP) 1 else 0)))
       .union(parentsRDD.map((_, (0, 0))))
       .reduceByKey((x, y) => (x._1 + y._1, x._2 + y._2))
   }
 
   def nErrorPerIndiv: RDD[(String, (Int, Int))] = {
-    val indivRDD = sc.parallelize(trios.flatMap(t => Iterator(t.kid, t.dad, t.mom)).distinct)
+    val indivRDD = sc.parallelize(trios.flatMap(t => Iterator(t.kid, t.knownDad, t.knownMom)).distinct)
     mendelErrors
       .flatMap(_.implicatedSamplesWithCounts)
       .union(indivRDD.map((_, (0, 0))))
       .reduceByKey((x, y) => (x._1 + y._1, x._2 + y._2))
   }
 
-  def writeMendel(filename: String, tmpDir: String) {
-    val sampleIdsBc = sc.broadcast(sampleIds)
-    mendelErrors.map(_.toLineMendel(sampleIdsBc.value))
-      .writeTable(filename, tmpDir, Some("FID\tKID\tCHR\tSNP\tCODE\tERROR"))
+  def mendelKT(): KeyTable = {
+    val signature = TStruct(
+      "fid" -> TString,
+      "s" -> TString,
+      "v" -> TVariant,
+      "code" -> TInt,
+      "error" -> TString)
+
+    val rdd = mendelErrors.map { e => Row(e.trio.fam.orNull, e.trio.kid, e.variant, e.code, e.errorString) }
+
+    KeyTable(hc, rdd, signature, Array())
```

Comment:
Oh!  Then s, v?  There's one per kid and variant, right?  Seems like a natural key to me.